configuration/config-txt.md

```diff
@@ -379,6 +379,31 @@ Your HDMI monitor may support only a limited set of formats. To find out which f
 
 The `edid.dat` should also be provided when troubleshooting problems with the default HDMI mode.
 
+### Custom mode
+
+If your monitor requires a mode that is not in one of the tables above, then it is possible to define a custom [CVT](http://en.wikipedia.org/wiki/Coordinated_Video_Timings) mode for it instead:
+
+    hdmi_cvt=<width> <height> <framerate> <aspect> <margins> <interlace> <rb>
+
+| Value | Default | Description |
+| --- | --- | --- |
+| width | (required) | width in pixels |
+| height | (required) | height in pixels |
+| framerate | (required) | framerate in Hz |
+| aspect | 3 | aspect ratio 1=4:3, 2=14:9, 3=16:9, 4=5:4, 5=16:10, 6=15:9 |
+| margins | 0 | 0=margins disabled, 1=margins enabled |
+| interlace | 0 | 0=progressive, 1=interlaced |
+| rb | 0 | 0=normal, 1=reduced blanking |
+
+(Fields at the end can be omitted to use the default values.)
+
+Note that this simply creates the mode (group 2 mode 87); in order to make the Pi use this by default you must add some additional settings.  As an example, the following selects an 800x480 resolution and enables audio drive:
```

Comment:
Just a stylistic thing, but would this look better as "...this simply *creates* the mode..." ? (added italics)